packages/user/src/components/LoginWrapper.tsx

```diff
@@ -1,30 +1,37 @@
+import { AppConfig } from "@dzangolab/react-config";
 import { useTranslation } from "@dzangolab/react-i18n";
 import { FC, useState } from "react";
+import { Link } from "react-router-dom";
 import { toast } from "react-toastify";
 
 import LoginForm from "./LoginForm";
-import { useConfig, useUser } from "../hooks";
+import { useUser } from "../hooks";
 import { verifySessionRoles } from "../supertokens/helpers";
 import login from "../supertokens/login";
 
 import type { LoginCredentials, SignInUpPromise } from "../types";
 
+import { ROUTES } from "@/constants";
+
 interface IProperties {
+  appConfig?: AppConfig;
   handleSubmit?: (credential: LoginCredentials) => void;
   onLoginFailed?: (error: Error) => void;
   onLoginSuccess?: (user: SignInUpPromise) => void;
   loading?: boolean;
+  showLinks?: boolean;
 }
 
 export const LoginWrapper: FC<IProperties> = ({
+  appConfig,
```

Comment:
instead of complex and big object as appConfig, lets use simple boolean and string props. We want to keep this component as simple as possible and reusable in simple places where appConfig may not be necessary.

Reply:
eg. 

- showSignupLink
- signupRoute (default: ROUTES.SIGNUP)